conf/app.routes

```diff
@@ -1,11 +1,9 @@
 # microservice specific routes
 
-# TODO add endpoints for field definitions CRUD operations
```

Comment:
We will need the endpoints for handling the fields definitions.
But we can think about them later :)

Reply:
we removed TODO based on your previous feedback